.percy.js

```diff
@@ -11,6 +11,12 @@ module.exports = {
       /(.*)\/views\/beta\/(.*)\.html/,
     ],
     exclude: ['site-template/*'],
+    overrides: [
+      {
+        files: /(.*)\/components\/rss-feeds\/index\.html/,
+        waitForSelector: 'table[data-table-name="rss-feed"]',
```

Comment:
Is this related to the yarn migration or rather a fix that just happened to end up here?

Reply:
A bit of both. I noticed it's the same error we get e.g. here https://github.com/commercetools/commercetools-docs-kit/pull/1145